### PR TITLE
Feature/AddsFabricFramework

### PR DESCRIPTION
**Pull request by cordhernandez**
#154

### CODE REVIEW

File: BlackNectar/BlackNectar/AppDelegate.swift [hunk 1]
```diff
@@ -21,6 +23,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
         
+        Fabric.with([Crashlytics.self, Answers.self])
```
Comment (SirWellington) [inline BlackNectar/BlackNectar/AppDelegate.swift:26]: Nice.

File: BlackNectar/BlackNectar/AppDelegate.swift [hunk 2]
```diff
@@ -49,6 +54,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         return true
         
     }
+    
+    func logUser() {
+        
+        Crashlytics.sharedInstance().setUserIdentifier(UIDevice.current.name)
```
Comment (SirWellington) [inline BlackNectar/BlackNectar/AppDelegate.swift:60]: I don't know if this is a really useful identifier, but better than nothing I guess.

**Review by SirWellington — APPROVED** [review]
I don't really like that we are checking in Fabric's code into the Repository, but I don't really see a better way at this point.

File: BlackNectar/BlackNectar/AppDelegate.swift (hunk 1 above)
Comment (cordhernandez) [reply BlackNectar/BlackNectar/AppDelegate.swift:26]: ty

File: BlackNectar/BlackNectar/AppDelegate.swift (hunk 2 above)
Comment (cordhernandez) [reply BlackNectar/BlackNectar/AppDelegate.swift:60]: just felt like writing

**Comment by cordhernandez** [comment]
This branch has been approved by admin SirWellington. Merging then deleting branch.

**Pull request merged**